Makefile

```diff
@@ -25,34 +28,58 @@ unlink:
 
 .PHONY: lint
 lint:
+	npm --prefix es-bs-base run lint
+# TODO: set up linting for es-ds-components and es-ds-docs
```

Comment:
is this remaining linting captured in a ticket somewhere?

Reply:
[Yup](https://energysage.atlassian.net/browse/CED-1726)! Not sure of the status on that one